docs/8-tilgangsnivå.adoc

```diff
@@ -40,6 +40,6 @@ Formålet med kategorien begrenset offentlighet er kunne skille ut datasett som
 
 Det kreves særskilt rettslig grunnlag for å få tilgang til datasettet. Typiske eksempler er gradert informasjon, sensitive personopplysninger eller forretningshemmeligheter. Aktuelle grunnlag for utlevering kan være samtykke fra den/de som har rett på vern, særhjemler eller partsinnsyn (partenes rett til å gjøre seg kjent med opplysninger i egen sak).
 
-Formålet med kategorien unntatt offentlighet er kunne skille ut data som kun kan deles på særskilt rettslig grunnlag.
+Formålet med kategorien unntatt offentlighet er å kunne skille ut data som kun kan deles på særskilt rettslig grunnlag.
 
-Uansett hvordan du klassifiserer datasettet, er det viktig å spesifisere hvilken hjemmel som ligger til grunn for innsamlingen og bruken av datasettet. Vurderingen går også her ut på om deler av et begrenset datasett likevel kan gjøres tilgjengelig som “grønne” data. Det som klassifiseres som rødt, skal likevel beskrives og beskrivelsene tilgjengeliggjøres i Felles datakatalog. footnote:[Registreringsløsning for Felles datakatalog: https://registrering-fdk.ppe.brreg.no/loggin  Publiseringsløsning for Felles datakatalog: https://fellesdatakatalog.brreg.no/] Behovet for å vite hva begrensningene for bruken av et datasett er, er viktig for den som skal vurderer å ta i bruk et datasett. Derfor bør du også henvise til den lovformuleringen med hjemmel og formål som regulerer bruken av datasettet, for eksempel i Lovdata. Se ellers https://doc.difi.no/data/spesifikasjon-for-beskrivelse-av-formal-og-hjemmel/[spesifikasjon for beskrivelse av formål og hjemmel.]
+Uansett hvordan du klassifiserer datasettet, er det viktig å spesifisere hvilken hjemmel som ligger til grunn for innsamlingen og bruken av datasettet. Vurderingen går også her ut på om deler av et begrenset datasett likevel kan gjøres tilgjengelig som “grønne” data. Det som klassifiseres som rødt, skal likevel beskrives og beskrivelsene tilgjengeliggjøres i Felles datakatalog. footnote:[Publisere i Felles datakatalog:https://data.norge.no/publishing] Behovet for å vite hva begrensningene for bruken av et datasett er, er viktig for den som skal vurderer å ta i bruk et datasett. Derfor bør du også henvise til den lovformuleringen med hjemmel og formål som regulerer bruken av datasettet, for eksempel i Lovdata. Se ellers https://doc.difi.no/data/spesifikasjon-for-beskrivelse-av-formal-og-hjemmel/[spesifikasjon for beskrivelse av formål og hjemmel.]
```

Comment:
"footnote:[Publisere i Felles datakatalog:https://data.norge.no/publishing]": Space manglet rett foran https.

Reply:
Angående "Uansett hvordan du klassifiserer datasettet, er det viktig å spesifisere hvilken hjemmel som ligger til grunn for innsamlingen og bruken av datasettet. ": 
Jeg synes setningen er uklar. Sier vi her at man også skal definere hjemmel for både innsamling og bruk åpne og offentlige data? Jeg husker ikke helt nøyaktig hva DCAT-AP-NO sier om dette, men i FDK er det for eksempel ikke mulig å registrere hjemmel for datasett dersom tilgangsnivå er offentlig. Dersom tilgangsnivå er "Begrenset offentlighet" eller "Unntatt offentlighet" kan man registrere skjermingshjemmel, behandlingsgrunnlag og utleveringshjemmel. Jeg lurer derfor på hvordan setningen skal forstås. Jeg er også i tvil om fargekoder fortsatt skal brukes. Jeg finner ikke fargekoder i DCAT-AP-NO og jeg kan heller se at de er i bruk som symboler i FDK portal eller som angivelse av tilgangsnivå i FDK registreringsløsning. Skal vi da fortsatt bruke betegnelser som "grønne", "gule" og "røde" data?